client/components/App.jsx

```diff
@@ -1,49 +1,50 @@
+import React from "react";
+import "../assets/stylesheets/style.scss";
+import { Route, Switch, BrowserRouter as Router } from "react-router-dom";
+import Header from "./header/Header";
+import Home from "./home/Home";
+import RegisterUser from "./auth/RegisterUser";
+import LoginUser from "./auth/LoginUser";
+import AdminLogin from "./auth/AdminLogin";
+import Page404 from "./Page404";
+import AdminDashboard from "./adminDashboard/AdminDashboard";
+import ContentList from "./content/ContentList";
+import NewContentForm from "./content/NewContentForm";
+import Students from "./students/Students";
+import PendingApprovals from "./adminDashboard/PendingApprovals";
+import ContentFeedback from "./content/ContentFeedback";
+import Content from "./content/Content";
+import Onboarding from "./auth/Onboarding";
+import StudentDashboard from "./students/studentDashboard/StudentDashboard";
 import EditContent from './content/EditContent';
 
-class App extends React.Component {
-  render() {
-    return (
-      <>
-        <Router>
-          <Header />
-          <Switch>
-            <Route exact path="/" component={Home} />
-            <Route path="/register" component={RegisterUser} />
-            <Route path="/login" component={LoginUser} />
-            <Route path="/admin/login" component={AdminLogin} />
-            <Route exact path="/admin/dashboard" component={AdminDashboard} />
-            <Route path="/content/list" component={ContentList} />
-            <Route path="/content/new" component={NewContentForm} />
-            <Route path="/students" component={Students} />
-            //TODO:students/all?
-            <Route path="/pendingapprovals" component={PendingApprovals} />
-            <Route path="/content/:contentid" component={Content} />
-            {/* <Route path="/:deliveryid" component={ContentFeedback} /> */}
-            <Route path="/feedback/:deliveryid" component={ContentFeedback} />
-            <Route path="/editcontent" component={EditContent} />
-            <Route component={Page404} />
-          </Switch>
-        </Router>
-      </>
-    );
-  }
+function App() {
+  return (
+    <>
+      <Router>
+        <Header />
+        <Switch>
+          <Route exact path="/" component={Home} />
+          <Route exact path="/register" component={RegisterUser} />
+          <Route path="/login" component={LoginUser} />
+          <Route exact path="/admin/login" component={AdminLogin} />
+          <Route exact path="/admin/dashboard" component={AdminDashboard} />
+          <Route exact path="/content/list" component={ContentList} />
+          <Route exact path="/content/new" component={NewContentForm} />
+          <Route exact path="/admin/students" component={Students} />
+          <Route exact path="admin/pendingapprovals" component={PendingApprovals} />
+          <Route exact path="/content/:contentid" component={Content} />
+          <Route exact path="/feedback" component={ContentFeedback} />
```

Comment:
change to `            <Route path="/feedback/:deliveryid" component={ContentFeedback} />
`